client/components/data/query-ticket-support-configuration/index.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ticketSupportConfigurationRequest,
+} from 'state/ticket-support/configuration/actions';
+
+class QueryTicketSupportConfiguration extends Component {
+	componentWillMount() {
+		this.props.ticketSupportConfigurationRequest();
+	}
```

Comment:
It might be useful to check `isRequesting` here to avoid doubling up on requests.

For example, if the `<QueryTicketSupportConfiguration />` component is used in two places at once, they'll both fire off the same request when its only necessary once. Using an `if( ! isRequesting )` check here can avoid that.